cmd/subnetcmd/import.go

```diff
@@ -3,282 +3,28 @@
-overwrite an existing subnet with the same name. To allow overwrites, provide the --force
-flag.`,
-	}
-	cmd.Flags().BoolVarP(
-		&overwriteImport,
-		"force",
-		"f",
-		false,
-		"overwrite the existing configuration if one exists",
-	)
-	cmd.Flags().StringVar(
-		&repoOrURL,
-		"repo",
-		"",
-		"the repo to import (ex: ava-labs/avalanche-plugins-core) or url to download the repo from",
-	)
-	cmd.Flags().StringVar(
-		&branch,
-		"branch",
-		"",
-		"the repo branch to use if downloading a new repo",
-	)
-	cmd.Flags().StringVar(
-		&subnetAlias,
-		"subnet",
-		"",
-		"the subnet configuration to import from the provided repo",
-	)
-	return cmd
-}
-
-func importSubnet(cmd *cobra.Command, args []string) error {
-	if len(args) == 1 {
-		importPath := args[0]
-		return importFromFile(importPath)
-	}
-
-	if repoOrURL == "" && branch == "" && subnetAlias == "" {
-		fileOption := "File"
-		apmOption := "Repository"
-		typeOptions := []string{fileOption, apmOption}
-		promptStr := "Would you like to import your subnet from a file or a repository?"
-		result, err := app.Prompt.CaptureList(promptStr, typeOptions)
-		if err != nil {
-			return err
-		}
-
-		if result == fileOption {
-			return importFromFile("")
-		}
-	}
-
-	// Option must be APM
-	return importFromAPM()
-}
-
-func importFromFile(importPath string) error {
-	var err error
-	if importPath == "" {
-		promptStr := "Select the file to import your subnet from"
-		importPath, err = app.Prompt.CaptureExistingFilepath(promptStr)
-		if err != nil {
-			return err
-		}
-	}
-
-	importFileBytes, err := os.ReadFile(importPath)
-	if err != nil {
-		return err
-	}
-
-	importable := models.Exportable{}
-	err = json.Unmarshal(importFileBytes, &importable)
-	if err != nil {
-		return err
-	}
-
-	subnetName := importable.Sidecar.Name
-	if subnetName == "" {
-		return errors.New("export data is malformed: missing subnet name")
-	}
-
-	if app.GenesisExists(subnetName) && !overwriteImport {
-		return errors.New("subnet already exists. Use --" + forceFlag + " parameter to overwrite")
-	}
-
-	err = app.WriteGenesisFile(subnetName, importable.Genesis)
-	if err != nil {
-		return err
-	}
-
-	err = app.CreateSidecar(&importable.Sidecar)
-	if err != nil {
-		return err
-	}
-
-	ux.Logger.PrintToUser("Subnet imported successfully")
-
-	return nil
-}
-
-func importFromAPM() error {
-	installedRepos, err := apmintegration.GetRepos(app)
-	if err != nil {
-		return err
-	}
-
-	var repoAlias string
-	var repoURL *url.URL
-	var promptStr string
-	customRepo := "Download new repo"
-
-	if repoOrURL != "" {
-		for _, installedRepo := range installedRepos {
-			if repoOrURL == installedRepo {
-				repoAlias = installedRepo
-				break
-			}
-		}
-		if repoAlias == "" {
-			repoAlias = customRepo
-			repoURL, err = url.ParseRequestURI(repoOrURL)
+		Use:   "import",
+		Short: "Import subnets into avalanche-cli",
+		Long: `The subnet command suite provides a collection of tools for developing
```

Comment:
This will require an update to the https://docs.avax.network/subnets/reference-cli-commands page

Reply:
https://github.com/ava-labs/avalanche-docs/pull/1096